locales/Update de_DE 2800-2900.json

```diff
@@ -2814,62 +2814,62 @@
+                "no-wallet-name": "Kein Walletname",
+                "no-wallet-address": "Keine Walletadresse"
               }
             }
           },
           "destination-address": {
-            "label": "Empfängeradresse",
+            "label": "Ziel Adresse",
             "placeholder": "Nur {{currency}} an diese Adresse senden",
             "errors": {
               "invalid-wallet": "Ungültige Adresse"
             }
           },
           "movement": {
-            "error": "Bitte warten Sie, bis Ihre letzte Ethereum-Überweisung abgeschlossen ist.",
-            "pending-error": "Bitte warten Sie, bis Ihre ausstehende Ethereum-Überweisung abgeschlossen ist.",
+            "error": "Bitte warte, bis deine letzte Ethereum-Überweisung abgeschlossen ist.",
+            "pending-error": "Bitte warte, bis deine ausstehende Ethereum-Überweisung abgeschlossen ist.",
             "token-requires-approval": "Ein Zuschlag von {{ amount }} {{ currency }} ist erforderlich, damit dieser Token in den Trading Contract überwiesen werden kann."
           },
           "transfer-amount": {
             "label": "Menge",
             "network-transaction-fee": "maximale Transaktionsgebühr",
-            "loading-fee": "Loading fee…",
+            "loading-fee": "Lade Gebühr…",
             "errors": {
-              "gas-shortage": "Sie benötigen mindestens {{amount}} {{currencySymbol}} auf Ihrem persönlichen Konto, um Transaktionsgebühren zu bezahlen.",
-              "fee-shortage": "You must save {{amount}} ETH for the Ethereum network fee.",
-              "fee-shortage-trading": "You need {{amount}} ETH in your personal account for Ethereum network fees."
+              "gas-shortage": "Du benötigst mindestens {{amount}} {{currencySymbol}} auf auf deinem persönlichen Konto, um Transaktionsgebühren zu bezahlen.",
+              "fee-shortage": "Du benötigst mindestens {{amount}} ETH für die Ethereum Transaktionsgebühr.",
+              "fee-shortage-trading": "Du benötigst mindestens {{amount}} ETH in deinem persönlichen Konto für die Ethereum Transaktionsgebühren."
             }
           },
           "btc": {
-            "funds-not-confirmed": "Ihr Bitcoin-Guthaben erfordert sechs Netzbestätigungen, bevor Sie auf den Handelskontrakt übertragen können. Bitte warten Sie, bis die letzten Überweisungen bestätigt werden.",
-            "wait-until-external-transfer-completes": "Bitte warten Sie, bis Ihre letzte externe Überweisung abgeschlossen ist, bevor Sie eine Einzahlung vornehmen.",
-            "temporary-limit": "Ein- und Auszahlungen von Bitcoin müssen größer als 0,001 BTC sein, während wir diesen neuen Markt testen. Die Beschränkung wird in naher Zukunft aufgehoben werden."
+            "funds-not-confirmed": "Ihr Bitcoin-Guthaben erfordert sechs Bestätigungen des Bitcoin Netwerks, bevor es auf dein Trading Contract übertragen werden kann. Bitte warte, bis die letzten Überweisungen bestätigt werden.",
```

Comment:
Bindestrich vor "Netzwerks".

Finde die maschinelle Überseztung etwas schlimm… Kann man verbessern?

Reply:
"Deine Bitcoin Transaktion  erfordert sechs Bestätigungen des Bitcoin Netzwerks, bevor es in deinen Trading Contract ankommt. Bitte warte, bis die letzte Transaktion bestätigt wurde."   Vll so?